index.html

```diff
@@ -5174,7 +5174,6 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 						<td class="role-related">
 							<ul>
 								<li><rref>list</rref></li>
```

Comment:
When there's only one of something in the characteristics table, the `ul` `li` list structure is removed.
So the only thing in the `<td class="role-related">` should be `<rref>list</rref>`.

Reply:
Updated! Though I might point out that this doesn't seem to be a consistent practice elsewhere in the document, including the single-list-item `role-parent` row in the same block that I reference for this one :)

A quick search shows 45 other single-item lists within tables on the page -- I'd be happy to de-list all of them, if that's desired.

Reply:
> A quick search shows 45 other single-item lists within tables on the page -- I'd be happy to de-list all of them, if that's desired.

Oh man. Maybe someday, but not today. :)